test/jdk/javax/swing/JSlider/bug6524424.java

```diff
@@ -52,15 +80,7 @@ public static void main(String[] args) {
         frame.setContentPane(panel);
         frame.setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);
```

Comment:
`frame.setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE); ` redundant can be removed.